packit_service/worker/handlers/bodhi.py

```diff
@@ -96,6 +102,12 @@ def pre_check(self) -> bool:
                 f"Bodhi update configured only for '{configured_branches}'."
             )
             return False
+        if self.koji_build_event.owner != self.expected_build_owner:
+            logger.debug(
+                f"Build {self.koji_build_event.build_id} submitted "
+                f"by {self.koji_build_event.owner}, skipping."
+            )
+            return False
```

Comment:
for Koji builds, we already do the filtering in the fedmsg [here](https://github.com/packit/packit-service-fedmsg/blob/3883a95f0f67ff56ffa3ad61f3a9822b9b7c3fcb/packit_service_fedmsg/consumer.py#L131)

Reply:
Nice! so I can drop that commit.